resources/js/components/RoomShareButton.vue

```diff
@@ -115,8 +115,10 @@ const roomUrl = computed(() => {
 });
 
 const formattedAccessCode = computed(() => {
-  return String(props.room.access_code)
-    .match(/.{1,3}/g)
-    .join("-");
+  return isNaN(props.room.access_code)
```

Comment:
That would format 6 digit numeric codes with a dash.
However when entering the 6 digit code there is no dash input mask